deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -81,14 +81,9 @@ objects:
           - "--cluster_name=hawkular-metrics"
           - "--data_volume=/cassandra_data"
           - "--internode_encryption=all"
-          - "--require_node_auth=true"
+          - "--require_node_auth=false"
```

Comment:
Then we have a problem :) The certs generated by OpenShift cannot be used as client certificates, as the feature flag on the cert is explicitly disabled. So, either we use the certs generated by OpenShift and not use it for client auth, or we don't use OpenShift certs (the purpose of this PR).